src/__test__/components/Header/LayerRoutes.spec.tsx

```diff
@@ -0,0 +1,274 @@
+import React, { useEffect, useRef, useState } from 'react';
```

Comment:
This file doesn't make sense, please take a look at this (https://testing-library.com/docs/react-testing-library/api/#renderhook-options) documentation to see how to test a hook

Reply:
Please, let me know if I'm correct in my affirmation @enzoggqs, Are you trying to test the hooks?